src/py/mat3ra/made/tools/build/passivation/helpers.py

```diff
@@ -0,0 +1,127 @@
+from collections import Counter
+from typing import List, Dict
+
+from mat3ra.esse.models.materials_category_components.entities.core.zero_dimensional.atom import AtomSchema
+
+from mat3ra.made.material import Material
+from mat3ra.made.tools.analyze.material import MaterialWithCrystalSites
+from mat3ra.made.tools.build import MaterialWithBuildMetadata
+from mat3ra.made.tools.build.defect.point.builders import AtomAtCoordinateConfiguration
+from mat3ra.made.tools.build.passivation.analyzer import (
+    SurfacePassivationMaterialAnalyzer,
+    CoordinationBasedPassivationMaterialAnalyzer,
+)
+from mat3ra.made.tools.build.passivation.builders import PassivationBuilder
+from mat3ra.made.tools.build.passivation.configuration import PassivationConfiguration
+
+
+def create_passivated_surface(
```

Comment:
Maybe just `passivate_surface`?? We give it a slab and the function does the passivation and returns a material